helpdesk_mgmt/controllers/myaccount.py

```diff
@@ -65,7 +65,8 @@ def portal_my_tickets(
         searchbar_filters = {
             "all": {"label": _("All"), "domain": []},
         }
-        for stage in request.env["helpdesk.ticket.stage"].search([]):
+        stages = self._get_applicable_stages(HelpdeskTicket.team_id, False)
```

Comment:
`HelpdeskTicket.team_id` is always going to be False, as there's no recordset. Why using this weird form?